build.sbt

```diff
@@ -45,18 +47,18 @@ lazy val magnum = project
     Test / fork := true,
     publish / skip := false,
     libraryDependencies ++= Seq(
-      "org.scalameta" %% "munit" % "0.7.29" % Test,
+      "org.scalameta" %% "munit" % "1.0.2" % Test,
       "com.dimafeng" %% "testcontainers-scala-munit" % testcontainersVersion % Test,
       "com.dimafeng" %% "testcontainers-scala-postgresql" % testcontainersVersion % Test,
-      "org.postgresql" % "postgresql" % "42.6.0" % Test,
+      "org.postgresql" % "postgresql" % "42.7.4" % Test,
       "com.dimafeng" %% "testcontainers-scala-mysql" % testcontainersVersion % Test,
-      "mysql" % "mysql-connector-java" % "8.0.32" % Test,
-      "com.h2database" % "h2" % "2.1.214" % Test,
+      "mysql" % "mysql-connector-java" % "8.0.33" % Test,
+      "com.h2database" % "h2" % "2.3.232" % Test,
       "com.dimafeng" %% "testcontainers-scala-oracle-xe" % testcontainersVersion % Test,
-      "com.oracle.database.jdbc" % "ojdbc11" % "21.9.0.0" % Test,
+      "com.oracle.database.jdbc" % "ojdbc11" % "23.5.0.24.07" % Test,
       "com.dimafeng" %% "testcontainers-scala-clickhouse" % testcontainersVersion % Test,
-      "com.clickhouse" % "clickhouse-jdbc" % "0.4.1" % Test classifier "http",
-      "org.xerial" % "sqlite-jdbc" % "3.41.0.0" % Test
+      "com.clickhouse" % "clickhouse-jdbc" % "0.6.0" % Test classifier "http",
```

Comment:
Looks like there's a 0.7.0 version now

Reply:
Yeah, I tried to update to 0.7.0 but it generates this error:
<img width="858" alt="Screenshot 2024-10-11 at 10 08 38 AM" src="https://github.com/user-attachments/assets/425c6dac-c6b9-4f23-aaa0-68c00d42097c">

Not sure where it comes from nor how to fix it. That's why I kept 0.6.0

Reply:
Awesome, thanks for checking